docs/index.md

```diff
@@ -258,14 +261,47 @@ to reflect your cluster configuration. Refer to the [kraken configuration](https
 and [configuration examples](https://github.com/startxfr/tekton-catalog/blob/stable/task/kraken-scenario/0.1/samples/) 
 for details on how to configure theses resources.
 
-#### Start as a single taskrun
+2) Start as a single taskrun
 
 ```bash
 oc apply -f https://github.com/startxfr/tekton-catalog/raw/stable/task/kraken-scenario/0.1/samples/taskrun.yaml
 ```
 
-#### Start as a pipelinerun
+3) Start as a pipelinerun
 
 ```yaml
 oc apply -f https://github.com/startxfr/tekton-catalog/raw/stable/task/kraken-scenario/0.1/samples/pipelinerun.yaml
 ```
+
+4) Deploying kraken using a helm-chart
```

Comment:
I'm not super clear about the tekton pipeline vs this installation type. Think this does the same thing as the above 3 steps? Not sure

Reply:
Right, it's just an alternate way of deploying and running Kraken.

Reply:
Having these as numbers confused me to thinking that they were steps but they are independent from eachother, correct? Maybe use a bullet or sub title

Reply:
Right, they are independent from each other, will add a bullet.

Reply:
Done. Thanks.